watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/GLSD_anaerobic_digester/GLSD_anaerobic_digestion_global_costing.yaml

```
@@ -0,0 +1,46 @@
+base_currency: USD_2020
+base_period: year
+defined_flows:
+  electricity:
+    value: 0.08
+    units: USD_2020/kWh
+global_parameters:
+  plant_lifetime:
+    value: 20
+    units: year
+  utilization_factor:
+    value: 0.9
+    units: dimensionless
+  land_cost_percent_FCI:
+    value: 0
+    units: dimensionless
+  working_capital_percent_FCI:
+    value: 0
+    units: dimensionless
+  salaries_percent_FCI:
+    value: 0
+    units: 1/year
+  benefit_percent_of_salary:
+    value: 0
+    units: dimensionless
+  maintenance_costs_percent_FCI:
+    value: 0.02
+    units: 1/year
+  laboratory_fees_percent_FCI:
+    value: 0
+    units: 1/year
+  insurance_and_taxes_percent_FCI:
+    value: 0
+    units: 1/year
+  wacc:
+    # Weighted Average Cost of Capital (WACC)
+    value: 0.08
```

```suggestion
    value: 0.05
```
Did the team specify this to be 0.08? If not, and this was based on the last PR for 1690, that PR had some adjusted values based on what the team specified. We shouldn't use those adjusted values here though unless 1669 specified to do so.

---

I think for most of the flowsheets, we have electricity cost as 0.08 $/kWh with the bound from 0.05 to 0.12, I followed the convention, and this is also the value I indicate in the template.

---

The default value for plant lifetime is 30 years, for utilization factor it is 0.85, and for WACC it is 0.05. @adam-a-a is right that the values you have (20, 0.9, and 0.08, respectively) come from the 1690 flowsheet, but that team explicitly provided us with the non-default parameter values.